includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -63,6 +65,7 @@ public function getData() {
 						'type' => 'translatable-text',
 						'key' => 'global-navigation-wikis-header',
 					],
+					'tracking_label' => 'link.wikis',
 				],
 				'links' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "links" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)